apps/server/src/modules/task/mapper/task.mapper.ts

```diff
@@ -1,5 +1,6 @@
 import { TaskWithStatusVo } from '@shared/domain';
 import { TaskResponse } from '../controller/dto';
+import { TaskStatusMapper } from './task-status.mapper';
```

Comment:
Should we add for each task mapping a additional mapper file, or put different methodes to the existing?

Reply:
I would prefer more concrete named files